substrate/frame/broker/src/adapt_price.rs

```diff
@@ -17,58 +17,97 @@
 
 #![deny(missing_docs)]
 
-use crate::CoreIndex;
+use crate::SaleInfoRecord;
 use sp_arithmetic::{traits::One, FixedU64};
-use sp_runtime::Saturating;
+use sp_runtime::{FixedPointNumber, FixedPointOperand, Saturating};
+
+/// Performance of a past sale.
+#[derive(Copy, Clone)]
+pub struct SalePerformance<Balance> {
+	/// The price at which the last core was sold.
+	///
+	/// Will be `None` if no cores have been offered.
+	pub sellout_price: Option<Balance>,
+
+	/// The base price (lowest possible price) that was used in this sale.
+	pub price: Balance,
```

Comment:
Naming nitpicking but min_price makes 100x more sense imo. 

> The meaning of BASE PRICE is the basic cost of something without adding anything extra. 

In that context that price is not base price since target (middle of sellout) is the new proper base price. This should be renamed to min_price

Reply:
Sigh. Just changed everything to base price. To me the name made sense as it is the base of the leadin curve. But I can change it again if people agree it is a better name.

Reply:
Ah I just saw your reply @Overkillus because this was marked as solved, I wondered why it had been changed again everywhere. The reason I advocated for `base_price` is that it is the basis to which all multipliers and discounts are applied. `min_price` can only be a minimum, but depending on the implementation this is not necessarily the case. I think this name change to `min_price` is a regression, but I don't see the value in flipping back and forth on just readability so I'll leave it up to you @eskimor. I just think in a pallet where readability is already an issue making things more confusing should be avoided.